datadog_checks_dev/datadog_checks/dev/tooling/commands/metadata.py

```diff
@@ -0,0 +1,138 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import csv
+
+import click
+
+from .utils import (
+    CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success, echo_waiting, echo_warning
+)
+from ..constants import get_root
+
+from ..utils import load_manifest, get_valid_checks
+from ...utils import file_exists, dir_exists, resolve_path
+
+METADATA_FILE = 'metadata.csv'
+
+REQUIRED_HEADERS = {
+    'metric_name',
+    'metric_type',
+    'description',
```

Comment:
@l0k0ms @zippolyte is there a limit of characters for `description`?

Reply:
If there is one, I don't know about it